packages/contractkit/CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+# Changelog
+All notable changes to the [Celo ContractKit package](https://www.npmjs.com/package/@celo/contractkit) will be documented in this file. 
+
+This package will follow the release process outlined [here](https://docs.celo.org/community/release-process).
```

Comment:
Should we update this to the Smart Contracts release process instead of just the release process menu? https://docs.celo.org/community/release-process/smart-contracts

Reply:
Hm, but contractkit would follow this release process right (the one on this PR), not the the smart contract one necessarily.